README.md

```diff
@@ -0,0 +1,58 @@
+The main reason why we use Linux is to have a similar environment to the Raspberry Pi.
+Another reason why we use Linux is because our software only works on Linux right now.
+We recommend two options if you do not have Linux already.
+
+# Dual booting [Highly Recommended] 
+
+## Background Knowledge 
+--------------- 
+Dual booting will keep 2 operating systems on your machine (only one operating system will be running on your machine at a time), so when you first start your computer, you can choose which operating system you want to run on.
+We highly recommend dual booting since you can utilize 100% of your computer's hardware for only one operating system(using a Virtual Machine, shares the PC hardware between operating systems; see VM Section).
+
+Pros | Cons
```

Comment:
I feel like a good con is that it's possible to brick your computer (or wipe away an entire OS) if you mess up.....  I think that should be a bolded warning LOL